qa/tasks/ceph_manager.py

```diff
@@ -1179,20 +1179,25 @@ def raw_cluster_cmd_result(self, *args, **kwargs):
         proc = self.controller.run(**kwargs)
         return proc.exitstatus
 
-    def run_ceph_w(self):
+    def run_ceph_w(self, channel=None):
         """
         Execute "ceph -w" in the background with stdout connected to a StringIO,
         and return the RemoteProcess.
-        """
-        return self.controller.run(
-            args=["sudo",
-                  "daemon-helper",
-                  "kill",
-                  "ceph",
-                  '--cluster',
-                  self.cluster,
-                  "-w"],
-            wait=False, stdout=StringIO(), stdin=run.PIPE)
+
+        :param channel: Specifies the channel to be watched. This can be
+                        'cluster', 'audit', ...
+        """
+        args = ["sudo",
+                "daemon-helper",
+                "kill",
+                "ceph",
+                '--cluster',
+                self.cluster,
+                "-w"]
+        if channel is not None:
+            args.append("--watch-channel")
```

Comment:
As the `ceph` argument is called `--watch-channel`, what about renaming the method parameter to `watch_channel`?

Reply:
Done